wordpress.org/public_html/wp-content/themes/pub/wporg-support-2024/inc/block-config.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Set up configuration for dynamic blocks.
+ */
+
+namespace WordPressdotorg\Forums\Block_Config;
+
+/**
+ * Actions and filters.
+ */
+add_filter( 'render_block_context', __NAMESPACE__ . '\wporg_render_block_context', 10, 3 );
+add_filter( 'wporg_ratings_data', __NAMESPACE__ . '\wporg_set_rating_data', 10, 2 );
+
+/**
+ * Update ratings blocks with real rating data.
+ *
+ * @param array $data    Rating data.
+ * @param int   $post_id Current post.
+ *
+ * @return array
+ */
+function wporg_set_rating_data( $data, $post_id ) {
+	$post = wporg_support_get_compat_object();
+
+	if ( ! class_exists( '\WPORG_Ratings' ) ) {
+		return '';
```

Comment:
This function should return an array — the empty string does work since the filter value is run through `wp_parse_args`, but to match the function docs it should be an array.

```suggestion
		return [];
```

Reply:
Actually, this could also return $data, but that might be less clear?

```suggestion
		return $data;
```